### ISSUE TITLE
Astropy deprecation warning in conversions.py

### ISSUE THREAD

**Issue by bwvdnbro** [SWIFTSIM/swiftsimio#135]
I am getting the following warnings when using `swiftsimio` with `astropy 5.1`:
```
WARNING: AstropyDeprecationWarning: `astropy.cosmology.flrw.a_B_c2` is a private variable (since v5.1) and in future will raise an exception. [astropy.cosmology.flrw]
WARNING: AstropyDeprecationWarning: `astropy.cosmology.flrw.critdens_const` is a private variable (since v5.1) and in future will raise an exception. [astropy.cosmology.flrw]
```
Unfortunately, I couldn't find another way to obtain these variables through the API. I'll just signal it here, so that someone more knowledgeable about `conversions.py` can look at this.

**Comment by bwvdnbro** [comment]
Why was this closed? #139 has not been merged and I still get this warning when I run the latest version of swiftsimio.

**Comment by MatthieuSchaller** [comment]
Sorry, I misread the message above that said "fixed #135".
